DemoApp/CustomMessageContentView.swift

```diff
@@ -22,7 +22,7 @@ class CustomMessageContentView: ChatMessageContentView {
             }
         }
         
-        guard let authorNameLabel = authorNameLabel, authorNameLabel.text != "" else {
+        guard let authorNameLabel = authorNameLabel, let authorText = authorNameLabel.text, authorText.isEmpty else {
```

Comment:
This also works and is a bit shorter and easier to read 👍 
```suggestion
        guard let authorNameLabel = authorNameLabel, authorNameLabel.text?.isEmpty == false else {
```

Reply:
Yeah, I considered this one but I preferred the readability of the one I implemented, wdyt?